Deliverables/D2. Requirements.md

```diff
@@ -33,9 +33,13 @@ For both office workers and gamers who have too much free time on their hands, G
 ## 4. Non-functional requirements
 - Secrets
 - Power ups
+- Portability
+- Avaliability
+- Useability
 
 ## 5. MVP
-
+- working start menu that connects to to the first level via a play button
+- 3 levels of golf with their own working scoring and continues onto to the next level
```

Comment:
I would do full sentences for MVP. 
ex:
For developing the minimum viable product (MVP) for Golf-N-Stuff, our strategy is to develop three levels of the miniature golf courses. The features we would implement would be our functional requirements. By implementing these, it will help us develop our MVP. We want to focus on the mechanics of the game, ease of use, and statistics for scoring.